### DISCUSSION TITLE
💑 Bumpkin Relationships

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#2203] [Ideas]
In Sunflower Land, players will have the opportunity to interact and build relationships with a variety of Bumpkins. By performing favours, such as cooking good food, players can strengthen their relationship with the Bumpkins and receive valuable rewards in return.

## Relationship Metrics

The player's relationship with each Bumpkin will be tracked and displayed in the form of a relationship meter. The higher the meter, the stronger the relationship and the better the rewards received from the Bumpkin.

Each Bumpkin will have their own unique personality and preferences, making each relationship unique and challenging. Players will have to figure out what each Bumpkin likes and dislikes in order to build a strong relationship.

Certain Bumpkins may also favour characteristics of your Bumpkin - for instance, a Blacksmith NPC would favour a Bumpkin who has crafted 500 Pickaxes.

## Rewards

As players strengthen their relationships with Bumpkins, they will receive rare items or wearables as rewards. These NFTs will be unique, collectible and can be traded or sold within the game.

These rewards can be received at random by visiting daily and talking to a Bumpkin. Some extremely rare rewards may occur on an off chance.

**Quest Feature**

This proposal would replace the current Quest system in game. Instead of completing activity challenges, players would need to ful-fill the relationship metrics to attain the item.

For instance, to attain the Fruit Picker Apron a player must now do a combination of the following:

- Collect lots of fruit
- Provide them with Fruit Juices
- Cook Apple Pie for them
- Talk with them 5 days in a row.

## Bumpkin Personalities

We are looking to introduce 10-12 unique Bumpkins throughout Sunflower Land that you can interact with. Each Bumpkin will have its own clothing and personality. Each one will be an NFT on the Blockchain with custom wearables.

Some Bumpkins will be accessible straight away, others are reserved for islands at higher levels for players.

Examples include:

- Blacksmith Bumpkin
- Greedy Goblin Bumpkin
- Rich Sunflorian Bumpkin
- Aunt Betty Bumpkin
- Lumberjack Bumpkin

As always we are looking for feedback and suggestions - especially for the types of Bumpkins ❤️ 🤗

**Comment by SarielWang93** [comment]
I don't quite understand how to ensure the rarity and uniqueness of the reward NFTs.

**Reply by adamhannigan** [reply]
Good point - We will need to develop a clearer tutorial and or docs around certain Bumpkins and what rewards they can provide 😄

**Comment by Galphath** [comment]
I love this idea! I think we will ned a grandma bumpkin *wink*

**Comment by drumdrumdrum3337** [comment]
Master Zen Bumpkin - loves koans, rewards patience and dedication in others
Proposition Joe Bumpkin - loves deals (get me this from that bumpkin, then I give you what you want)
Pirate Bumpkin - keep his drinks coming and he'll tell you where to find stuff (has just a few teeth left so mostly speaks through the parrot on his shoulder)
Joker Bumpkin - loves to pull one's leg, so sometimes nothing comes out of interaction with him (apart from a good laugh), but sometimes a trump or two might fall off from his sleeve
Serious Accountant Bumpkin - doesn't like to mess around, everything should be precise and exact. Be a minute late or one potato short, and he won't be happy
God Complex Bumpkin - loves shiny gifts and when produce is burnt in his name, might reward those praising him

**Reply by drumdrumdrum3337** [reply]
Shiva Bumpkin - destroys and rebuilds his piece of land every night. One day his land can contain 6 beds, next day beds are gone and there a couple of mines, then some trees, etc. Can let a bumpkin use what's available on that land, as long as the bumpkin brings their own seeds, axes, etc.

**Comment by rodolfoddias** [comment]
As someone who played Harvest Moon a lot, I really LOVE this concept. I would also add a "marriage" feature, so we could bond with some of the Bumpkins for extra farm buffs and unlock new possibilities such as having a child Bumpkin :)

The Bumpkin Plaza could have different suitors, each one with an unique set of skills that would help our Bumpkin farmers once we marry them. After we reach max level of affinity with them, we would be able to propose using a DIAMOND RING (really hard to obtain).

Different Bumpkins NPC would have different buffs, so we would have to choose accordingly to our strategies. Different Bumpkins could marry different NPCs, adding another layer to the Multiple Bumpkin Features.  What do you guys think?
